tests/mock-data.js

```diff
@@ -25,10 +26,13 @@ mock.exampleDocWithInvocation.beta =
   require('./mock-documents/example-doc-with-beta-invocation');
 
 const didContext = require('./mock-documents/did-context');
-_loaderData['https://w3id.org/did/v0.11'] = didContext;
+_loaderData.set('https://w3id.org/did/v0.11', didContext);
```

Comment:
```suggestion
_loaderData.set('https://w3id.org/did/v1', didContext);
```

We should stop using the v0.11 context everywhere... Veres One has been upgraded to did/v1. I expect more changes will be required here.

Reply:
Yeah, not doing those changes right now.